ONSdigital/eq-author-app
# Ear 2004 new data tabs

### What is the context of this PR?

PR contains the first step towards implementing the pre-pop data. This change rebrands 'metadata' as 'sample file data' in the UI, and moves it to a new 'Data' nav option, whilst removing the existing 'Metadata' option. Via vertical tabs it also adds a new tab/page for 'ONS dataset'. This new page is implemented as text only, with functionality to be added in future tickets.

The code includes a move and partial renaming of the existing 'metadata' page code. However as the data is still refered to as 'metadata' on the backend, in general only the frontend files and functions have been renamed.

The new ONS dataset tab will only appear when feature flag of 'dataset' is present

Note, although there is a high 'files changed' count, many are due to relocation/deletion/renaming.

### How to review

1. Ensure that the existing 'Metadata' tab option is removed, and replace with a 'Data' option
2. Ensure that the current url of qid/metadata redirects to qid/data/sample-file-data, to allow any bookmarks etc to still function.
3. Ensure that the rebranded 'Metadata' page (now 'Sample file data') works as before (add, remove, remove all etc)
4. Ensure the rebranded page does not mention 'metadata' in the UI via any of the methods of usage
5. Ensure that the new ONS dataset tab appears, and displays the required text.
